packages/fury-adapter-oas3-parser/test/unit/parser/oas/parseServersArray-test.js

```diff
@@ -0,0 +1,77 @@
+const { Fury } = require('fury');
+const { expect } = require('../../chai');
+const parse = require('../../../../lib/parser/oas/parseServersArray');
+const Context = require('../../../../lib/context');
+
+const { minim: namespace } = new Fury();
+
+describe('#parseHostsObject', () => {
```

Comment:
```suggestion
describe('#parseServersArray', () => {
```

To match the function we are testing.

Reply:
changed